source/main.bmx

```diff
@@ -2857,9 +2857,9 @@ Type TSaveGame Extends TGameState
 		New TGameState.Initialize()
 
 		Local savegameEventData:TData = new TData
-		savegameEventData.AddString("saveName", saveURI)
-		savegameEventData.AddInt("saved_savegame_version", loadedSaveGameVersion)
-		savegameEventData.AddInt("current_savegame_version", TSaveGame.SAVEGAME_VERSION)
+		savegameEvenTData.Add("saveName", saveURI)
```

Comment:
evenTData ->eventData? or eventTData?

Reply:
Guter Fund. Autoreplace :)

Reply:
war die Änderung in der Variable beabsichtigt?

Reply:
Du meinst des Variablennamens? Nein. Das war ein "Find and replace" auf `TData.AddString("` zu `TData.Add("` ...